server/world/tx.go

```
@@ -12,18 +11,69 @@ import (
+		f(ctx)
+		return nil
+	})
 }
 
-// Range returns the lower and upper bounds of the World that the Tx is
+// Range returns the lower and upper bounds of the World that the Context is
 // operating on.
-func (tx *Tx) Range() cube.Range {
-	return tx.w.ra
+func (ctx *Context) Range() cube.Range {
+	return ctx.tx.w.ra
```

golangci-lint (2.12.2)

[error] QF1008: could remove embedded field "tx" from selector

(staticcheck)